src/python/WMCore/MicroService/Unified/RequestInfo.py

```diff
@@ -293,7 +254,7 @@ def getMulticore(request):
         mcores.extend([int(v) for v in mcoresCol.values()])
     return max(mcores)
 
-def getSiteWhiteList(request, siteInfo, reqSpecs=None, pickone=False, verbose=True):
+def getSiteWhiteList(request, siteInfo, campaignInfo, reqSpecs=None, pickone=False, verbose=True):
```

Comment:
what is campaignInfo parameter looks like? Is new class CampaignConfig or something else?

Reply:
It is CampaignConfig class which is the dictionary. I think it works with get but not parameters I will update the code